ipython_profiles/profile_s6bm/startup/03-plans.py

```diff
@@ -183,6 +185,7 @@ def scan_closure():
         yield from bps.mv(det.cam.frame_type, 2)
 
         # 1-8 take the back white
+        yield from bps.mv(det.proc1.num_filter, n_frames)
```

Comment:
Similar, can these two lines be combined?

Reply:
A more compact/concise syntax will be used once we are confident about the required sequence for setting different parameters.